cli/src/commands/invoke/remote.rs

```diff
@@ -22,7 +24,17 @@ pub async fn invoke(
         console::style(&display_path).underlined().bold()
     );
 
-    println!("{}\n", console::style(function.url().await?).dimmed());
+    let url = if url_path.is_empty() {
+        function.url().await?.replace(['{', '}', '+', '*'], "")
```

Comment:
@quesabe please add a comment about why we need this transformation in place. Also, I'm not sure I understand this condition `if url_path.is_empty()`. Can you explain please?